### PR TITLE
feat: add `ignoreMissing` option to allow skip valid language checking

### PR DESCRIPTION
**Pull request by HashimotoYT**
This PR adds `ignoreMissing` option what allows skipping checking for valid language names.

/cc @zeke @sarahs for GH docs review side

### CODE REVIEW

**Comment by HashimotoYT** [comment]
@zeke These changes introduced to fix the randomly broken code blocks in `electron-i18n` in translate documents. The broken code blocks very hard to track and fix immediately, this option helps to disable the checking for valid language and fix the build on our CI processes.

File: test/fixtures/unknown-language.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Let's create the own language
+
+```the-unknown-language-whats-actually-graphql
```
Comment (zeke) [inline test/fixtures/unknown-language.md:3]:
```suggestion
```some-unknown-language
```

File: test/index.js [hunk 2]
```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
     file.content.should.include('<em>Markdown</em>')
   })
 
+  describe('ignoreMissing', () => {
+    it('throw an error when unknown language is present', () => {
+      return hubdown(fixtures.unknownLanguage).should.to.be.rejectedWith(/Unknown language: `the-unknown-language-whats-actually-graphql`/)
```
Comment (zeke) [inline test/index.js:73]:
```suggestion
      return hubdown(fixtures.unknownLanguage).should.to.be.rejectedWith(/Unknown language: `some-unknown-language`/)
```

File: test/index.js [hunk 3]
```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
...
+    })
+
+    it('should work when ignoreMissing on the scene', async () => {
+      const file = await hubdown(fixtures.unknownLanguage, { ignoreMissing: true })
+      $ = cheerio.load(file.content)
+      fixtures.unknownLanguage.should.include('```the-unknown-language-whats-actually-graphql')
```
Comment (zeke) [inline test/index.js:79]:
```suggestion
      fixtures.unknownLanguage.should.include('```some-unknown-language')
```

File: test/index.js [hunk 4]
```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
...
+      $('pre > code.hljs.language-the-unknown-language-whats-actually-graphql').length.should.equal(1)
```
Comment (zeke) [inline test/index.js:80]:
```suggestion
      $('pre > code.hljs.language-some-unknown-language').length.should.equal(1)
```

File: test/index.js (hunk 2 above)
Comment (zeke) [inline test/index.js:73]:
```suggestion
      return hubdown(fixtures.unknownLanguage).should.be.rejectedWith(/Unknown language: `the-unknown-language-whats-actually-graphql`/)
```

File: index.js [hunk 5]
```diff
@@ -73,15 +73,21 @@ function makeHash (markdownString, opts) {
   return hasha(markdownString + optsString)
 }
 
-function createProcessor (before) {
+function createProcessor (before, ignoreMissing) {
```
Comment (zeke) [inline index.js:76]: Hmm it seems like one of my comments got lost.  I think this `ignoreMissing` should be renamed for clarity. What do you think about allowing a `highlight` option to be passed instead?

```js
{
  highlight: {
    ignoreMissing: true
  }
}
```

Comment (HashimotoYT) [reply index.js]: @zeke 4ef1dece829864a730700319494b9ee2df742131 introduces the `highlight` object which can accept all options from [rehype-highlight](https://github.com/rehypejs/rehype-highlight#options) module

File: test/index.js [hunk 6]
```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
     file.content.should.include('<em>Markdown</em>')
   })
 
+  describe('ignoreMissing', () => {
+    it('throw an error when unknown language is present', () => {
```
Comment (zeke) [inline test/index.js:72]:
```suggestion
    it('throws an error when unknown language is present', () => {
```

File: test/index.js [hunk 7]
```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
     file.content.should.include('<em>Markdown</em>')
   })
 
+  describe('ignoreMissing', () => {
```
Comment (zeke) [inline test/index.js:71]:
```suggestion
  describe('highlight.ignoreMissing option', () => {
```

File: test/index.js [hunk 8]
```diff
@@ -65,6 +68,19 @@ describe('hubdown', () => {
...
+    it('throw an error when unknown language is present', () => {
+      return hubdown(fixtures.unknownLanguage).should.be.rejectedWith(/Unknown language: `some-unknown-language`/)
+    })
+
+    it('should work when ignoreMissing on the scene', async () => {
```
Comment (zeke) [inline test/index.js:76]:
```suggestion
    it('should work when the highlight.ignoreMissing option is true', async () => {
```

**Pull request merged**